lte/gateway/python/integ_tests/s1aptests/test_attach_he_policy.py

```diff
@@ -0,0 +1,239 @@
+        print(
+            "********************** Sending RAR for IMSI",
+            "".join([str(i) for i in req.imsi]),
+        )
+        he_domain1 = "192.168.128.1"
+        assert utils.count_record_of_imsi_to_domain(imsi, he_domain1) == 0
+        self._sessionManager_util.create_ReAuthRequest(
+            "IMSI" + "".join([str(i) for i in req.imsi]),
+            policy_id1,
+            flow_list1,
+            qos1,
+            he_urls=HeaderEnrichment(urls=[he_domain1]),
+        )
+
+        response = self._s1ap_wrapper.s1_util.get_response()
+        self.assertEqual(
+            response.msg_type, s1ap_types.tfwCmd.UE_ACT_DED_BER_REQ.value
+        )
+        act_ded_ber_req_oai_apn = response.cast(
+            s1ap_types.UeActDedBearCtxtReq_t
+        )
+        self._s1ap_wrapper.sendActDedicatedBearerAccept(
+            req.ue_id, act_ded_ber_req_oai_apn.bearerId
+        )
+
+        print("Sleeping for 5 seconds")
+        time.sleep(5)
+        # Send PDN Connectivity Request
+        apn = "ims"
+        self._s1ap_wrapper.sendPdnConnectivityReq(ue_id, apn)
+        # Receive PDN CONN RSP/Activate default EPS bearer context request
+        response = self._s1ap_wrapper.s1_util.get_response()
+        self.assertEqual(
+            response.msg_type, s1ap_types.tfwCmd.UE_PDN_CONN_RSP_IND.value
+        )
+        act_def_bearer_req = response.cast(s1ap_types.uePdnConRsp_t)
+        addr = act_def_bearer_req.m.pdnInfo.pAddr.addrInfo
+        sec_ip = ipaddress.ip_address(bytes(addr[:4]))
+
+        print(
+            "********************** Sending Activate default EPS bearer "
+            "context accept for UE id ",
+            ue_id,
+        )
+
+        print("Sleeping for 5 seconds")
+        time.sleep(5)
+
+        dl_flow_rules = {
+            default_ip: [flow_list1],
+        }
+        # 1 UL flow is created per bearer
+        num_ul_flows = 2
+        # Verify if flow rules are created
+        self._s1ap_wrapper.s1_util.verify_flow_rules(
+            num_ul_flows, dl_flow_rules
+        )
+        print("*********** Moving UE to idle mode")
+        print(
+            "************* Sending UE context release request ",
+            "for UE id ",
+            ue_id,
+        )
+        # Send UE context release request to move UE to idle mode
+        req = s1ap_types.ueCntxtRelReq_t()
+        req.ue_Id = ue_id
+        req.cause.causeVal = gpp_types.CauseRadioNetwork.USER_INACTIVITY.value
+        self._s1ap_wrapper.s1_util.issue_cmd(
+            s1ap_types.tfwCmd.UE_CNTXT_REL_REQUEST, req
+        )
+        response = self._s1ap_wrapper.s1_util.get_response()
+        self.assertEqual(
+            response.msg_type, s1ap_types.tfwCmd.UE_CTX_REL_IND.value
+        )
+
+        print("Sleeping for 5 seconds")
+        time.sleep(5)
+
+        # Verify if paging flow rules are created
+        ip_list = [default_ip, sec_ip]
+        self._s1ap_wrapper.s1_util.verify_paging_flow_rules(ip_list)
```

Comment:
Do we need to do this in this test? Here and in a few other places we're verifying quite a few non he things

Reply:
I kept the validations that are copied from other tests to confirm additional HE parameter in rule does not change existing flows and attach procedure.